eth/polar/polaris.go

```diff
@@ -147,8 +172,7 @@ func (pl *Polaris) StartServices() error {
 	pl.filterSystem = utils.RegisterFilterAPI(pl.stack, pl.backend, &defaultEthConfig)
 
 	go func() {
-		// TODO: unhack this.
-		time.Sleep(2 * time.Second) //nolint:gomnd // we will fix this eventually.
+		time.Sleep(3 * time.Second) //nolint:gomnd // TODO:fix, this is required for hive...
```

Comment:
is this only required for hive now? then should be a test config flag that only sleeps if test mode

Reply:
handle in later pr